recipes/r-knitcitations/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = '1.0.8' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-knitcitations
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/knitcitations_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/knitcitations/knitcitations_{{ version }}.tar.gz
+  sha256: 3b366d6b7cd02998d451f820a7883e6c5c315391a5301112a80c8b725969081e
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+
+  host:
+    - r-base
+    - r-refmanager >=0.8.2
+    - r-digest
+    - r-httr >=0.3
+
+  run:
+    - r-base
+    - r-refmanager >=0.8.2
+    - r-digest
+    - r-httr >=0.3
+
+test:
+  commands:
+    - $R -e "library('knitcitations')"           # [not win]
+    - "\"%R%\" -e \"library('knitcitations')\""  # [win]
+
+about:
+  home: https://github.com/cboettig/knitcitations
+  license: MIT
+  summary: Provides the ability to create dynamic citations in which the bibliographic information
+    is pulled from the web rather than having to be entered into a local database such
+    as 'bibtex' ahead of time. The package is primarily aimed at authoring in the R
+    'markdown' format, and can provide outputs for web-based authoring such as linked
+    text for inline citations.  Cite using a 'DOI', URL, or 'bibtex' file key.  See
+    the package URL for details.
+  license_family: MIT
+
+extra:
+   recipe-maintainers:
+     - vivekbhr
```

Comment:
Could you add a few more maintainers?

    - johanneskoester
    - bgruening
    - daler
    - jdblischak
    - cbrueffer
    - dbast